op-e2e/system/helpers/tx_helper.go

```diff
@@ -89,7 +89,7 @@ func defaultDepositTxOpts(opts *bind.TransactOpts) *DepositTxOpts {
 // Transaction options and expected status can be configured in the applyTxOpts function by modifying the supplied TxOpts
 // Will verify that the transaction is included with the expected status on l2Client and any clients added to TxOpts.VerifyClients
 func SendL2TxWithID(t *testing.T, chainID *big.Int, l2Client *ethclient.Client, privKey *ecdsa.PrivateKey, applyTxOpts TxOptsFn) *types.Receipt {
-	opts := defaultTxOpts()
+	opts := DefaultTxOpts()
```

Comment:
This change in export doesn't seem to be necessary, and it's also a modification of the underlying optimism code which may make this more difficult to maintain.

Reply:
Yes, it was done to implement `SendL2TxNoReceip`, but this [function is not needed](https://github.com/EspressoSystems/optimism-espresso-integration/pull/117#discussion_r2071751912) and will be removed.